compass_sdk/parser.py

```diff
@@ -66,7 +66,6 @@ def __init__(
         self.num_workers = num_workers
 
         self.metadata_config = metadata_config
-        logger.info(f"CompassParserClient initialized with parser_url: {self.parser_url}")
 
```

Comment:
Any reason for removing it?

Reply:
Based on the comment of the PR:

> This log appears every request, it is quite noisy

Having said that, I would question why you are creating a client every single request instead of reusing it across requests? @BeatrixCohere